### PR TITLE
Milestone 1: Awesome-Books with ES6

### PR DESCRIPTION
**Pull request by skabeo**
**In this pull request I created a book website with functionalities which includes;**

- Allows users to add a new book.
- Also users can delete a particular book;
- Users can switch between the book list section, the add book section and also the contact section.
- Users input are saved on their local storage and displays when there is any.
- Used ES6 syntax for functionality of the web site.

### CODE REVIEW

File: modules/app.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+        sections[j].style.display = 'none';
+      }
+      const target = this.getAttribute('href').substr(1);
+      document.getElementById(target).style.display = 'block';
+
+      for (let k = 0; k < navLinks.length; k += 1) {
+        navLinks[k].style.color = 'black';
+      }
+      this.style.color = 'blue';
+    });
```
Comment (didierganthier) [inline modules/app.js]: Well done overall 💯  but for several reasons you should consider turning this into an arrow function. Arrow functions introduce concise body syntax or implicit return. This allows the omission of the curly brackets and the `return` keyword. Implicit return is useful for creating succinct one-line operations in `map`, `filter`, and other common array methods. 🤓 You code could look like this then 👇🏽 

```
navLinks[i].addEventListener('click', (event) => {
    // You code goes here
});
```

File: modules/bookshelf.js [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+        this.remove(book);
+        this.render();
+      });
+      li.appendChild(deleteButton);
+      bookList.appendChild(li);
+
+      if (i % 2 === 0) {
+        li.style.backgroundColor = '#e9e9e9';
+      }
+    }
```
Comment (didierganthier) [inline modules/bookshelf.js:44]: [OPTIONAL] The work is awesome 👍🏽 like the books 😆 but in order to further improve it, I'd suggest adding a check to show another message if there are no books in the list. Thoughts? 🤔 

This is what you have so far 👇🏽 

Screenshot 2023-03-27 at 07 57 51

**Review by didierganthier — CHANGES_REQUESTED** [review]
Hi @skabeo ,

🎉 Good job so far! You're making great progress on this project, but there are still a few areas that need attention before we can move on to the next step.



## Highlights
- Descriptive PR ✔️ 
- Linters passing ✔️ 
- Correct GitHub flow ✔️ 

## Required Changes 🛠
- Please review the comments under the review for specific changes that still need to be made.

## Optional Suggestions 💡
- While these comments with the [OPTIONAL] prefix won't prevent the approval of this pull request, I strongly recommend taking them into account as they can improve your code.

🚀 Keep pushing forward, you're almost there! If you have any questions or need clarification on anything, feel free to leave a comment in the pull request thread and don't forget to tag me so I'll receive the notification.

**Please note**
- To avoid confusion and delays, please do not open a new pull request for re-reviews. Use the same pull request submitted for the first review, unless otherwise instructed.

------
As per the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561), you have a limited number of reviews per project. If you feel that the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s). 

Cheers and happy coding! 👏👏👏

**Review by thecodechaser — APPROVED** [review]
## STATUS: APPROVED 🟢

## Hi @skabeo👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight 🎉
- Highly organized code. ✔️
- No linter errors. :heavy_check_mark:
- GitHub flow is followed. :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good title and summary. ✔️



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**